packages/react-core/src/evm/providers/thirdweb-sdk-provider.tsx

```diff
@@ -147,10 +147,17 @@ const WrappedThirdwebSDKProvider = <
       }
     }
 
+    // TODO: find a better way to fix the type error
+    type ForcedChainType = {
+      rpc: string[];
+      chainId: number;
+      nativeCurrency: { symbol: string; name: string; decimals: 18 };
+    };
+
     const mergedOptions = {
       readonlySettings,
       ...sdkOptions,
-      chains: supportedChains,
+      supportedChains: supportedChains as any as ForcedChainType[],
```

Comment:
Nice catch. There are some mandatory fields in the Chain type that we don't have here. I'm worried this may cause other issues down the line if we assume we have those fields, e.g. `testnet`, `slug`, `shortName`, `chain`, `name`. Should we maybe do a `getChains(supportedChains)` where we fetch the actual Chain obj based on the chaindId?

Reply:
this should be fine for now